packages/fuels-signers/src/wallet.rs

```diff
@@ -1,14 +1,24 @@
 use crate::provider::{Provider, ProviderError};
 use crate::Signer;
 use async_trait::async_trait;
+use coins_bip32::{path::DerivationPath, Bip32Error};
+use coins_bip39::{English, Mnemonic, MnemonicError};
+use elliptic_curve::rand_core;
+use eth_keystore::KeystoreError;
 use fuel_crypto::{Message, PublicKey, SecretKey, Signature};
 use fuel_gql_client::client::schema::coin::Coin;
 use fuel_tx::{Address, AssetId, Input, Output, Receipt, Transaction, UtxoId, Witness};
 use fuels_core::errors::Error;
 use fuels_core::parameters::TxParameters;
+use rand::{CryptoRng, Rng};
+use std::path::Path;
+use std::str::FromStr;
 use std::{fmt, io};
 use thiserror::Error;
 
+const DEFAULT_DERIVATION_PATH_PREFIX: &str = "m/44'/60'/0'/0/";
```

Comment:
Is this actually a "path prefix"? BIP-44 just calls it a "path."

Reply:
This is a prefix because it's missing the last digit, the index one. A "0" is concatenated when used and it gives us a bit of flexibility to increment it if the user would, for instance, create many wallets with the same phrase but default prefix derivation path.

Reply:
Ah got it